flood_adapt/object_model/direct_impacts.py

```diff
@@ -517,7 +518,11 @@ def _create_infometrics(self, fiat_results_df) -> Path:
             ext = ""
 
         # Get options for metric configurations
-        metric_types = ["mandatory", "infographic", "additional"]
```

Comment:
If we're going to make more and more things optional (i.e. the tipping points could also be optional), a better way to then do this is by building up the list like:
```
metrics_types = ['mandatory", "additional"]
if site.infometrics:
  metrics_types += 'infometrics'
if site.tipping_points: Not yet but at some point
  metrics_types += 'tipping_points'
```

Reply:
I will change it like this